dev/test/bundle.ts

```diff
@@ -24,7 +24,7 @@ import {
 } from './util/helpers';
 import IBundleElement = firestore.IBundleElement;
 
-describe('Bundle Buidler', () => {
+describe.only('Bundle Buidler', () => {
```

Comment:
If you merge @thebrianchen's changes form the Node10 branch into your PR you will get a lint check that will catch this :)

Reply:
There is a lot changes from bulk writer too. I will do the merge later..